src/main.py

```diff
@@ -28,7 +28,9 @@
 try:
     handler = WXKeyboardHandler(uiManager.frame)
     handler.register_key("control+alt+space", uiManager.toggle_visibility)
-    handler.register_key("control+alt+r", windowManager.hide_currently_focused_window)
+    handler.register_key(
+        "control+alt+r", windowManager.hide_currently_focused_window)
+    handler.register_key("control+shift+f1", uiManager.openDocs)
```

Comment:
I don't like the fact that this is a global hotkey. We can ad a help button in the UI, but I don't feel like adding yet another global hotkey even when the program is not in focus just to open documentation is a good idea.